stop

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+clear
```

Comment:
Might just be, but I think I'd be a bit annoyed if the stop script cleared my screen every time. What if there was still something I needed?